kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -53,10 +53,10 @@
       <p class="login-text no-account">{{ $tr('noAccount') }}</p>
       <div id="btn-group">
         <router-link v-if="canSignUp" class="group-btn" :to="signUp">
-          <icon-button id="signup-button" :text="$tr('createAccount')" :primary="true"/>
+          <k-button :text="$tr('createAccount')" :primary="false"/>
         </router-link>
         <a class="group-btn" href="/learn">
-          <icon-button id="guest-access-button" :text="$tr('accessAsGuest')" :primary="false"/>
+          <k-button :text="$tr('accessAsGuest')" :primary="false"/>
```

Comment:
I wouldn't recommend changing the secondary button styles just because one instance on the login page. I think the fact that these secondary buttons are black on white here works in our advantage, as we do want sign in to be the primary action, and then create account + access as guest to stand out against everything else. Given that most, if not all secondary buttons are against canvas colors of #f9f9f9 and #ffffff within Kolibri, I think we are okay to keep the secondary colors as they are. If there are issues with the login page being too dark, I'd probably push towards eventually tweaking the login page bkg or look entirely.